AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-staff-member/add-staff-member.component.ts

```diff
@@ -14,6 +14,7 @@ import { Constants } from 'src/app/common/constants';
     styleUrls: ['./add-staff-member.component.css']
 })
 export class AddStaffMemberComponent extends AddParticipantBaseDirective implements OnInit, OnDestroy {
+    @Input() staffMemberLocator: string;
```

Comment:
Does this need to be an input? If it will always be 'staffMemberEmail' I would store that in here.

Reply:
removed and used constant